### PR TITLE
Create Python script to run e2e Argo Workflow

### PR DESCRIPTION
**Pull request by andreyvelich**
Fixes: https://github.com/kubeflow/katib/issues/1376.

I added script to run our AWS Argo Workflow in python instead of deprecated `ksonnet`.
I removed [Python Unit Test](https://github.com/kubeflow/katib/blob/master/test/scripts/v1beta1/python-tests.sh) from the Workflow.
We will use GitHub Action to run these tests: https://github.com/kubeflow/katib/issues/1601

/cc @gaocegege @johnugeorge

### CODE REVIEW

**Comment by andreyvelich** [comment]
Test seems to be working.
@gaocegege @johnugeorge Please take a look.
cc  @thunderboltsid

File: prow_config.yaml [hunk 1]
```diff
@@ -45,12 +46,9 @@ workflows:
...
       - test/e2e/v1beta1/*
-      - test/scripts/v1beta1/*
-      - test/suggestion/v1beta1/*
-      - test/workflows/*
       - manifests/v1beta1/*
       - scripts/v1beta1/*
       - prow_config.yaml
       - Makefile
-    params:
+    kwargs:
```
Comment (johnugeorge) [inline prow_config.yaml:53]: This got changed?

Comment (andreyvelich) [reply prow_config.yaml:53]: @johnugeorge Check here: https://github.com/kubeflow/testing/blob/master/py/kubeflow/testing/run_e2e_workflow.py#L12-L17.
It uses `kwargs` in Python scripts.

**Pull request merged**